toolchain/sem_ir/formatter.cpp

```diff
@@ -131,8 +143,14 @@ class FormatterImpl {
   // A scope in which output should be buffered because we don't yet know
   // whether to include it in the final formatted SemIR.
   struct TentativeOutputScope {
-    explicit TentativeOutputScope(FormatterImpl& f) : formatter(f) {
-      index = formatter.AddChunk(false);
+    explicit TentativeOutputScope(FormatterImpl& f, size_t parent_chunk)
```

Comment:
How about calling this `parent_chunk_index`?

Reply:
Done.